setup.py

```diff
@@ -46,7 +46,7 @@
         'boto3',
         'scikit-learn',
         'scipy==1.4.1',
-        'spacy[lookups]==3.0.0',
+        'spacy[lookups]<3.1.0,>=3.0.0',
```

Comment:
In the issue description, it says that scispacy which asks for `>3.0.1.`. Do you think just `> 3.0.1` wouldn't work?

Reply:
It would but I guess we should set an upper bound, should we say <4.0.0?

Reply:
Yeah, that'd be more relaxed!

Reply:
aa2dc2c